infra/Makefile

```
@@ -0,0 +1,117 @@
+.PHONY: help build run test test-coverage fmt lint migrate-up migrate-down migrate-status docker-up docker-down clean test-local docker-prod-up generate-secrets create-env
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)